src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -219,4 +223,96 @@ void searchWithHttpsUrl() throws Exception {
     void searchWithHttpsUrlNotTrimmed() throws Exception {
         assertEquals(Optional.of(sliceTheoremPaper), finder.performSearchById("https : // arxiv . org / abs / 1405 . 2249 "));
     }
+
+    @Test
+    @Override
+    public void authorSearch() throws FetcherException {
+        List<BibEntry> results = finder.performSearch("au:\"Tobias Diez\"");
+        assertFalse(results.isEmpty());
+        results.forEach(bibEntry -> {
+
+            String author = bibEntry.getField(StandardField.AUTHOR).orElse("");
+
+            // Some papers have other authors
+            Assertions.assertTrue(author.contains("Tobias Diez"));
+        });
+    }
+
+    /**
+     * @throws FetcherException TODO: find out the correct search parameter names
+     */
+    @Test
+    @Override
+    public void yearSearch() throws FetcherException {
+    }
+
+    /**
+     * @throws FetcherException TODO: find out the correct search parameter names
+     */
+    @Test
+    @Override
+    public void yearRangeSearch() throws FetcherException {
```

Comment:
JabRef convention: Always `throws Exception`. At tests, the concrete kind of Exception is not important

```suggestion
    public void yearRangeSearch() throws Exception {
```